index.js

```diff
@@ -86,14 +93,11 @@ function init() {
 	return ret;
 }
 
-defineProps(chalk, init());
+defineProps(Chalk.prototype, init());
 
-chalk.styles = ansiStyles;
-chalk.hasColor = hasAnsi;
-chalk.stripColor = stripAnsi;
-chalk.supportsColor = supportsColor;
+Chalk.prototype.styles = ansiStyles;
```

Comment:
The idea is to get the names exported so I guess I could just stick them on exported object. Would then change the reference to `this.supportsColor` to just use the local supportsColor as I'm guessing there is not really much use in being able to override that.